cobalt/devinfra/kokoro/build/linux/package.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""Packages Cobalt for Linux."""
+
+import os
+import shutil
+import sys
+import tempfile
+
+out_dir = sys.argv[1]
+package_dir = sys.argv[2]
+
+with tempfile.TemporaryDirectory() as tmp_dir:
+  base_dir = os.path.join(tmp_dir, 'cobalt-linux')
+  os.makedirs(base_dir)
+
+  shutil.copy2(os.path.join(out_dir, 'cobalt'), base_dir)
```

Comment:
Please at least move files into an array, so you don't copy-paste `shutil.copy`  and os.path.join all over

Reply:
+1

Please define an array at the top of the file and loop over here. This script is pretty short now but as it grows (which it will) this list should still be immediately in front of the reader.

Reply:
> Please at least move files into an array, so you don't copy-paste `shutil.copy` and os.path.join all over

After all this, not even a comment here:
https://github.com/youtube/cobalt/pull/4804/files#diff-bf121a8163433bd66f2e378e6f31183eea77dd30a66ce27382510588cc5ab869